tests/.pylintrc

```diff
@@ -143,7 +143,8 @@ disable=print-statement,
         missing-module-docstring,
         missing-function-docstring,
         no-self-use,
-        protected-access
+        protected-access,
+        too-many-arguments
```

Comment:
Not sure why we need this ? 🤔

Reply:
It is because of one of the linting rule saying we don't accept more than 5 arguments per function. It is a bit strict but I think it forces to break down big functions into smaller ones enforcing clean code good practices :)

Reply:
In the test, I sometimes need a lot of argument because of the `@mocks/@patches`.
Notice that I only edited the pylintrc from the tests folder, the rule is still active in `nestor_api`